SayTheirNames/Source/Controller/Home/Person/Views/PersonNewsTableViewCell.swift

```diff
@@ -51,6 +51,8 @@ class PersonNewsTableViewCell: UITableViewCell {
         return "\(Self.self)"
     }
     
+    private var identifier: String = "PersonNewsCollectionViewCell"
```

Comment:
custom identifiers aren't needed if you use methods from `Reusable` protocol which all cells already conform to